manifests/garden/deployment.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Why do we need a new deployment for this? We already have a deployment defined for `wego-app`.

Reply:
The way Garden works is that it doesn't allow to have the same file in both garden Modules. The specific way in which we embed wego-app manifests forces me to include this file in the container module. But also this file needs to be included in the Kubernetes module so it can apply it. I'm open to suggestions on how to address this in a different approach.

Reply:
Let me take a look and get back to you. There may be a way.

Reply:
This is fixed now